examples/06_voice/src/main.rs

```diff
@@ -109,23 +108,32 @@ command!(deafen(ctx, msg) {
     }
 });
 
-command!(join(ctx, msg, args) {
-    let connect_to = match args.single::<u64>() {
-        Ok(id) => ChannelId(id),
-        Err(_) => {
-            check_msg(msg.reply("Requires a valid voice channel ID be given"));
+command!(join(ctx, msg) {
+    let guild_id = match CACHE.read().guild_channel(msg.channel_id) {
```

Comment:
I'd advise just using [`Message::guild`](https://docs.rs/serenity/0.5.3/serenity/model/channel/struct.Message.html#method.guild) here, as displayed by the usage of `Guild` below.

Reply:
No, i meant in this way to simplify fetching the channel id:
```rust
let guild = match msg.guild() {
    Some(g) => g,
    None => {
        check_msg(msg.channel_id.say("Groups and DMs not supported"));

        return Ok(());
    },
};

let channel_id = guild
    .read()
    .voice_states.get(&msg.author.id)
    .map(|vs| vs.channel_id);
```